mumble: add page (#8750)
tldr-pages/tldr: pages/linux/mumble.md

# mumble

> Low-latency, high quality voice chat software.
> More information: <https://www.mumble.info>.

- Open Mumble:

`mumble`

- Open Mumble and immediately connect to a server:

`mumble mumble://{{username}}@{{example.com}}`

- Open Mumble and immediately connect to a password protected server:

`mumble mumble://{{username}}:{{password}}@{{example.com}}`

> - Mute or unmute yourself in running Mumble instance:

Reviewer: We try to avoid `you`/`your` in the examples, are we talking about the :microphone: ? or we can use `input audio` too.
```suggestion
- Mute/unmute the microphone in a running Mumble instance:
```

Author: @navarroaxel oh ok, I didn't know that you/your should be avoided, must've missed that in the guidelines :see_no_evil: 

So in general yes, it's about whatever is configured as audio input device, which is probably set to a microphone because that's what the software is for, talking to others. I honestly don't know if there are other use cases where you would set your input device to something different than a microphone.
The button you can push in the GUI to do the same is also a microphone BTW

I'm ok with your suggestion and applied it in [1970b4cb26da0bff1fb248224c59b68efcd9ea23](https://github.com/tldr-pages/tldr/pull/8750/commits/1970b4cb26da0bff1fb248224c59b68efcd9ea23)

`mumble rpc {{mute|unmute}}`

> - Mute or unmute yourself and the audio output of Mumble:

Reviewer: ```suggestion
- Mute/unmute the microphone and the audio output of Mumble:
```
